packages/suite-data/src/browser-detection/index.ts

```diff
@@ -98,7 +98,7 @@ window.addEventListener('load', () => {
     };
 
     const firefox = {
-        name: 'Firefox 78+',
+        name: 'Firefox 102+',
```

Comment:
It would be nice to use version from `supportedBrowsers` const to avoid those errors